tests/utils/Modifiers.sol

```diff
@@ -74,7 +74,11 @@ abstract contract Modifiers is EvmUtilsBase {
     }
 
     modifier whenCallerAdmin() {
-        setMsgSender(users.admin);
+        setMsgSender(admin);
+        _;
+    }
+
+    modifier whenCallerComptroller() {
```

Comment:
It should have `setMsgSender(address(comptroller));` to avoid setting `comptroller` every time this modifier is used.